segfault when specifying an unknown output format

```
$ footloose status node0 -o yaml
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x827768]

goroutine 1 [running]:
main.show(0xd6f9a0, 0xc000093890, 0x1, 0x3, 0x0, 0x0)
	/home/damien/go/src/github.com/weaveworks/footloose/show.go:40 +0x178
github.com/spf13/cobra.(*Command).execute(0xd6f9a0, 0xc000093860, 0x3, 0x3, 0xd6f9a0, 0xc000093860)
	/home/damien/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:762 +0x465
github.com/spf13/cobra.(*Command).ExecuteC(0xd71160, 0x0, 0x0, 0xd70320)
	/home/damien/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2ec
github.com/spf13/cobra.(*Command).Execute(...)
	/home/damien/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800
main.main()
	/home/damien/go/src/github.com/weaveworks/footloose/footloose.go:28 +0x2e
```

--- a/show.go
+++ b/show.go
@@ -33,6 +33,9 @@ func show(cmd *cobra.Command, args []string) error {
 		return err
 	}
 	formatter, err := cluster.GetFormatter(showOptions.output)
+	if err != nil {
+		return err
+	}
 	machines, err := c.Inspect(args)
 	if err != nil {
 		return err

